framework/include/wifi_manager/wifi_manager.h

```diff
@@ -23,6 +23,11 @@
  * @{
  */
 
+/**
+ * @file wifi_manager.h
+ * @brief Provides APIs for Wi-Fi Manager
+ */
```

Comment:
Don't you need to set group?

Reply:
I think, in the above, there is a { for group, so ingroup is not need.

Reply:
Oh ok. Then, why do you repeat brief?

Reply:
Let's move file option to above comments and remove duplicated brief.